scripts/prepare-ci-environment.sh

```diff
@@ -10,8 +10,8 @@ fi
 # Python plugin for Community and Ultimate Edition
 
 export CWD=$(pwd)
-export IJ_VERSION="191.6183.62"
-export IJ_BUILD_NUMBER="191.6183.62"
+export IJ_VERSION="192.5728.98"
```

Comment:
This is the build number if it's EAP, and the release name if it's released. so `2019.2` in this case.